content-security-policy/connect-src/connect-src-json-import-allowed.sub.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta http-equiv="Content-Security-Policy" content="connect-src 'self' http://{{host}}:{{ports[http][0]}}; script-src 'self' 'unsafe-inline';">
```

Comment:
isn't `http://{{host}}:{{ports[http][0]}}` the same as self?

Reply:
I think it's the same, at least when serving as `http` -- I copied it from various other tests in the same folder such as https://github.com/web-platform-tests/wpt/blob/master/content-security-policy/connect-src/connect-src-beacon-allowed.sub.html

Reply:
It's unfortunate that we cannot test it at the moment... But I would just drop this since I am pretty sure it does nothing more than 'self'.